packages/core/src/connectors/github/constant.ts

```diff
@@ -0,0 +1,2 @@
+export const AUTHORIZE_ENDPOINT = 'https://github.com/login/oauth/authorize';
```

Comment:
Authorize is a verb. shall we use authorization in general?

Reply:
also shall we align the constant naming convention over the language? i was using camelCase since it's just like a normal variable / constant, also the UPPER_SNAKE_CASE is acceptable. just need to align.
@logto-io/eng any ideas?

Reply:
`AuthorizationEndpoint` is OK for me.

Reply:
per offline discussion, aligned to `authorizationEndpoint`